src/com/nullbyte/quarrycraft/Main.java

```diff
@@ -23,211 +24,215 @@
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.block.Action;
-import org.bukkit.event.block.BlockPistonEvent;
 import org.bukkit.event.block.BlockPistonExtendEvent;
 import org.bukkit.event.block.BlockPistonRetractEvent;
 import org.bukkit.event.player.PlayerInteractEvent;
 import org.bukkit.event.player.PlayerJoinEvent;
-import org.bukkit.plugin.Plugin;
 import org.bukkit.plugin.java.JavaPlugin;
 import org.bukkit.scheduler.BukkitRunnable;
 
+import com.nullbyte.quarrycraft.commands.TrustCommand;
 
+public class Main extends JavaPlugin implements Listener {
 
+	public List<Quarry> quarries;
 
-
-public class Main extends JavaPlugin implements Listener {
-	public JavaPlugin plugin;
-	public ArrayList<Quarry> quarries;
-	
 	GuideBookGiver gbGiver;
-	
+
 	public static int quarryLimit = 5;
 	public static int maxQuarryWidth = 50;
 	public static int maxQuarryLength = 50;
 	public static boolean welcomeMessages = true;
-	public static long guideBookCooldown = 1000*60*120;
+	public static long guideBookCooldown = 1000 * 60 * 120;
 	public static boolean doWGProtection = true;
 	public static int maxWGY = 100;
 	public static int minWGY = 20;
 	public static boolean doGPProtection = true;
 	public static int maxGPY = 100;
 	public static int minGPY = 20;
-	
+
+	private int maxTrustedPlayers = 2;
+
 	public static ConfigurableMessages configurableMessages;
 
-	class guideCommand implements CommandExecutor{
+	private TrustCommand trustCommand;
 
+	class guideCommand implements CommandExecutor {
 		@Override
-		public boolean onCommand(CommandSender sender, Command cmd, String alias, String[] args) {
-			if(args.length == 1 && sender.isOp() && args[0].equals("reload")) {
+		public boolean onCommand(CommandSender sender, Command command, String alias, String[] args) {
+			if (args.length < 1)
+				return false;
+
+			switch (args[0]) {
+			case "reload": {
 				configurableMessages.loadMessages();
 				sender.sendMessage(configurableMessages.reloading1());
 				loadConfig();
 				sender.sendMessage(configurableMessages.reloading2());
 				return true;
 			}
-			
-			
-			if(!(sender instanceof Player))
+			case "guide": {
+				if (!(sender instanceof Player)) {
+					sender.sendMessage(translateColors("&cOnly players can use this command"));

```

Comment:
Wherever sending messages to the players, could you have that message in the configurable messages so that server owners can change the messages to say whatever they want. Eg the server may want to use a language other than english so they could translate the messages on their own